### PR TITLE
Make screenshots smaller

### PR DESCRIPTION
**Pull request by wolfs**

### CODE REVIEW

File: contents/css/build-cache-guide.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
 .screenshot img {
     box-shadow: 0 0 20px 1px rgba(0,0,0,0.2);
 }
+
+div.screenshot {
+    width: 90%;
+    margin: auto;
+}
```
Comment (lptr) [inline contents/css/build-cache-guide.css]: We should use `div.screenshot` or `.screenshot` in both of these places, no?

Comment (wolfs) [inline contents/css/build-cache-guide.css]: fixed.

File: contents/css/build-cache-guide.css [hunk 2]
```diff
@@ -1,3 +1,8 @@
 .screenshot img {
     box-shadow: 0 0 20px 1px rgba(0,0,0,0.2);
 }
+
+div.screenshot {
+    width: 90%;
+    margin: auto;
```
Comment (lptr) [inline contents/css/build-cache-guide.css]: This should only be for the left and right margins, as it breaks the bottom margin:

image

Comment (wolfs) [inline contents/css/build-cache-guide.css]: fixed. Anything else how to make it better?

**Pull request merged**